### PR TITLE
Add cron task

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 name: Dart
 
 on:
+  schedule:
+    # “At 00:00 (UTC) on Sunday.”
+    - cron: '0 0 * * 0'
```
Comment (devoncarew) [inline .github/workflows/build.yaml:6]: 👍 thu

**Review by devoncarew — APPROVED** [review]

File: .github/workflows/build.yaml (hunk 1 above)
Comment (kevmoo) [reply .github/workflows/build.yaml:6]: thu?

**Comment by pq** [comment]
@kevmoo: what's the benefit of a scheduled build?

**Comment by kevmoo** [comment]
> @kevmoo: what's the benefit of a scheduled build?

It helps us sanity check the whole ecosystem.

Maybe this package isn't touched in a month, we still want tests running against it w/ new SDKs and dependencies so we can find any regressions.

**Comment by pq** [comment]
Thanks!  I was actually about to answer my own question.  There have been times where this would have been useful for the linter.  Thanks for the inspiration!

File: .github/workflows/build.yaml (hunk 1 above)
Comment (devoncarew) [reply .github/workflows/build.yaml:6]: Hmm, not sure how those extra chars got added -

Comment (kevmoo) [reply .github/workflows/build.yaml:6]: 🤷

**Pull request merged**